Client/src/Models/Datasets/IVariableModel.ts

```diff
@@ -1,44 +1,37 @@
-export interface IVariableAndUnitModel {
-    xVariableName: string,
-    yVariableName: string,
-    xVariableUnits: string,
-    yVariableUnits: string
-}
-
 export interface IVariableModel {
     type: string,
     variableNames: string[],
     units: string[]
 }
 
 export const IWidthModel: IVariableModel = {
-    type: 'width',
+    type: 'initial pressure',
     variableNames: ['cell length', 'cell width', 'critical tube diameter', 'minimum tube diameter'],
-    units: ['mm', 'cm', 'm']
+    units: ['kPa', 'cm', 'm']
```

Comment:
We are hardcoding these? I thought these models would be returned from backend in a later ticket? Or are they temporary?

Reply:
umm, @lposo put them in for the units, i guess just to build the skeleton for the AxesControl, We will need to get back to this and cleanup in the next sprint when we are doing the units page, ima add a //todo